Grid/qcd/utils/WilsonLoops.h

```diff
@@ -177,25 +177,37 @@ template <class Gimpl> class WilsonLoops : public Gimpl {
   }
 
   //////////////////////////////////////////////////
-  // average over all x,y,z the temporal loop
+  // average Polyakov loop in any direction (spatial or temporal)
```

Comment:
This subtracts a little information from the original comment. Might be worth saying something like "average Polyakov loop in direction $\mu$ over all directions $\ne\mu$".